CrazyCanvas/Source/GUI/SavedServerGUI.cpp

```diff
@@ -0,0 +1,98 @@
+#pragma once 
+
+#include "GUI/SavedServerGUI.h"
+#include "GUI/Core/GUIApplication.h"
+
+#include "NoesisPCH.h"
+
+
+using namespace LambdaEngine;
+using namespace Noesis;
+
+
+SavedServerGUI::SavedServerGUI(const LambdaEngine::String& xamlFile) :
+	m_ItemCount(0)
+{
+
+}
+
+void SavedServerGUI::Init(ListBox* pListView)
+{
+	/*textBlock = *new ObservableCollection<TextBlock>();
+	tBlock = *new TextBlock("ChrilleBoi");
+	textBlock->Add(tBlock);
+	m_pSavedServerList->SetItemsSource(textBlock);*/
+	
+	m_pSavedServerList = *pListView;
+
+}
+
+SavedServerGUI::~SavedServerGUI()
+{
+}
+
+void SavedServerGUI::AddServerItem(Grid* pParentGrid, const char* pServerN, const char* pMapN, const char* pPing, bool isRunning)
+{
+	
+	Ptr<Grid> grid = *new Grid();
+
+	for (int i = 0; i < 4; i++)
+	{
+		Ptr<ColumnDefinition> pColumnDef = *new ColumnDefinition();
```

Comment:
```suggestion
		Ptr<ColumnDefinition> columnDef = *new ColumnDefinition();
```
Shared pointer? View first comment